ui/shortcutdialog/table.go

```diff
@@ -90,6 +103,56 @@ func (shortcutTable *ShortcutTable) SetShortcuts(shortcuts []*shortcuts.Shortcut
 			SetMaxWidth(1)
 		shortcutTable.table.SetCell(row, shortcutCellIndex, eventCell)
 
+		if *shortcutTable.vimMode != vim.Disabled {
+			normalKey := ""
+			insertKey := ""
+			visualKey := ""
+
+			// HACK
+			// Empty tcell EventKey corresponds to Ctrl-Space
```

Comment:
Why????

Reply:
Some shortcuts do not have an equivalent in some modes, so I need to leave them blank in the dialog. This only checks if the vim modifier is empty.